view-samples/property-card-view/README.md

```diff
@@ -0,0 +1,45 @@
+# Flashcards
```

Comment:
Please set the title of this sample in heading 1 of README.md.

Reply:
Fixed. Thanks